scvi/model/_autozi.py

```diff
@@ -190,63 +203,72 @@ def get_marginal_ll(
             )
             for tensors in scdl:
                 sample_batch = tensors[_CONSTANTS.X_KEY]
-                local_l_mean = tensors[_CONSTANTS.LOCAL_L_MEAN_KEY]
-                local_l_var = tensors[_CONSTANTS.LOCAL_L_VAR_KEY]
                 batch_index = tensors[_CONSTANTS.BATCH_KEY]
                 labels = tensors[_CONSTANTS.LABELS_KEY]
 
                 # Distribution parameters and sampled variables
-                inf_outputs, gen_outputs, losses = self.module.forward(tensors)
+                inf_outputs, gen_outputs, _ = self.module.forward(tensors)
 
                 px_r = gen_outputs["px_r"]
                 px_rate = gen_outputs["px_rate"]
                 px_dropout = gen_outputs["px_dropout"]
                 qz_m = inf_outputs["qz_m"]
                 qz_v = inf_outputs["qz_v"]
                 z = inf_outputs["z"]
-                ql_m = inf_outputs["ql_m"]
-                ql_v = inf_outputs["ql_v"]
                 library = inf_outputs["library"]
 
                 # Reconstruction Loss
-                current_dev = px_rate.device
                 bernoulli_params_batch = self.module.reshape_bernoulli(
                     bernoulli_params,
-                    batch_index.to(current_dev),
-                    labels.to(current_dev),
+                    batch_index,
+                    labels,
```

Comment:
did you test this on a GPU?

Reply:
no, I can try it out. Is it just running pytest w/ a GPU?/